core/sodasql/scan/measurement.py

```diff
@@ -8,13 +8,17 @@
 #  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
-from dataclasses import dataclass
+from dataclasses import dataclass, astuple
 from typing import Optional, List
 
 from sodasql.common.json_helper import JsonHelper
 from sodasql.scan.group_value import GroupValue
 
 
+def is_equal_or_both_none(left, right):
+    return (left == right) or (left is None and right is None)
```

Comment:
I think  `left == right` should suffice to test for both `None` .. is there a reason for `or (left ...)` ?

Reply:
works! so no need for this